assets/js/session/index.js

```diff
@@ -427,7 +427,9 @@ function handleDocumentMouseDown(hook, event) {
   }
 
   // Find the focusable element, if one was clicked
-  const focusableEl = event.target.closest(`[data-focusable-id]`);
+  const focusableEl = event.target.closest
+    ? event.target.closest(`[data-focusable-id]`)
+    : false;
```

Comment:
Here we handle `mousedown` and the target should always be an element, or did you reproduce the issue without this too?

Reply:
yes, i didn't need that to reproduce the problem